WordPress/Classes/ViewRelated/Stats/Referrer Details/ReferrerDetailsTableViewController.swift

```diff
@@ -57,3 +57,10 @@ private extension ReferrerDetailsTableViewController {
          ReferrerDetailsSpamActionRow.self]
     }
 }
+
+// MARK: - Types
+extension ReferrerDetailsTableViewController {
```

Comment:
I'm not sure if this is the right place for this since this is only used by the table cells and not used in the view controller at all. What do you think about moving it to WPStyleGuide?

Reply:
Better place then view controller definitely :)